build/src/proto_path_map.rs

```diff
@@ -0,0 +1,251 @@
+use std::iter;
+
+use prost_build::Service;
+
+/// Maps a fully-qualified Protobuf path to a value using path matchers.
+/// Original implementation: https://github.com/tokio-rs/prost/blob/5a2c7092964ac2eaaa516c61bcd48e3c66ea16b3/prost-build/src/path.rs
```

Comment:
Nit: I would make this line a module comment `//!` because it applies to the full file